dune/grid/common/ZoltanPartition.cpp

```diff
@@ -53,11 +54,24 @@ std::vector<int> zoltanGraphPartitionGridOnRoot(const CpGrid& cpgrid,
     Zoltan_Set_Param(zz, "RETURN_LISTS", "ALL");
     Zoltan_Set_Param(zz, "DEBUG_LEVEL", "3");
     Zoltan_Set_Param(zz, "CHECK_GRAPH", "2");
+    Zoltan_Set_Param(zz,"EDGE_WEIGHT_DIM","0");
+    Zoltan_Set_Param(zz, "OBJ_WEIGHT_DIM", "0");
     Zoltan_Set_Param(zz, "PHG_EDGE_SIZE_THRESHOLD", ".35");  /* 0-remove all, 1-remove none */
 
     bool pretendEmptyGrid = cc.rank()!=root;
+    std::shared_ptr<CombinedGridWellGraph> grid_and_wells;
 
-    Dune::cpgrid::setCpGridZoltanGraphFunctions(zz, cpgrid, pretendEmptyGrid);
+    if( eclipseState )
+    {
+        Zoltan_Set_Param(zz,"EDGE_WEIGHT_DIM","1");
```

Comment:
Not being familiar with Zoltan details, I presume this means that each graph edge has a single-number weight, and that we do not use this (no weighting) in the default (no eclipseState) case?

Would this be the place to put transmissibilities for weights?